Makefile

```
@@ -46,3 +46,15 @@ test-resolver: ## Run resolver tests
 test-pkg: ## Run pkg tests
 	cd pkg && make test
 
+.PHONY: serve-docs 
+serve-docs: ## Serve docs
+	@command -v mkdocs >/dev/null 2>&1 || { \
+	  echo "mkdocs not found - please install it (pip install mkdocs-material)"; exit 1; } ; \
+	mkdocs serve
+	
+.PHONY: build-docs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)